R/license_note.R

```diff
@@ -0,0 +1,79 @@
+#' Generate LICENSE.note file.
+#'
+#' LICENSE.note generated by this function lists dependent Rust packages infomation.
+#' To use this function, the [cargo-lincense](https://crates.io/crates/cargo-license) command must be installed.
+#' @param force Logical indicating whether to re-generating LICENSE.note if LICENSE.note is already exists.
+#' @inheritParams register_extendr
+#' @return No return value, called for side effects.
+#' @export
+#' @importFrom dplyr %>%
```

Comment:
I don't think this is needed since it is imported elsewhere 
see https://github.com/extendr/rextendr/blob/9efe3f20e19b9a62c300dd78f04c4dcb6f60a042/R/rextendr.R#L9